src/objectTranslations/Course_Offering_Schedule__c-en_US.objectTranslation

```diff
@@ -14,50 +14,53 @@
         <name>Course_Offering__c</name>
         <relationshipLabel><!-- Course Offering Schedules --></relationshipLabel>
     </fields>
+    <fields>
+        <help><!-- The time of day the Course Offering ends. --></help>
+        <label><!-- End Time --></label>
+        <name>End_Time__c</name>
+    </fields>
     <fields>
         <help><!-- The Facility associated with the Course Offering for the specified time and day of week. --></help>
         <label><!-- Facility --></label>
         <name>Facility__c</name>
         <relationshipLabel><!-- Course Offering Schedules --></relationshipLabel>
     </fields>
     <fields>
-        <help><!-- The Course Offering Schedule includes Friday. --></help>
```

Comment:
did we mean to remove these lines in the translation files?

Reply:
I think it's just a result of doing a metadata retrieve from the server. We removed these help texts when we released the object in 1.7x. I can revert them if you're worried about them and followup on it later.

Reply:
no big deal